db_populate.py

```diff
@@ -0,0 +1,224 @@
+# This file is used to populate the database(mainly for testing)
+from lemur.utility import *
+
+
+def populate_db():
```

Comment:
This is setting up fixtures for an integration test then. At some point it'd a good idea to make a directory for such things, and write integration tests that spin up the db, and then make requests against the running app to make sure that it is working as intended.